util.c

```diff
@@ -5202,15 +5202,8 @@ void util_boot_system()
     return;
   }
 
-  // sometimes you need it, sometimes not - see bsc#1076839
-  #if defined(__x86_64__)
-    #define KEXEC_OPT	" -s"
-  #else
-    #define KEXEC_OPT	""
-  #endif
-
   strprintf(&buf,
-    "kexec" KEXEC_OPT " -l '/mnt/%s' --initrd='/mnt/%s' --append='%s'",
+    "kexec -a -l '/mnt/%s' --initrd='/mnt/%s' --append='%s'",
```

Comment:
well, I check patch at https://bugzilla.suse.com/attachment.cgi?id=761277&action=diff and still I do not see that -a, see only -c and -s there.

Reply:
ah, now I see it when I read whole bug report. So I propose to:

1. mark in comment that -a option is not yet stable enough and can change in future
2. add to comment link to this bug report when we start asking why it is there.

Reply:
The sr's in the bug say 'add -a' option. And the kexec folks say this as well. There's really no point to argue here.

Reply:
Where does it say '-a' is unstable? I don't see the need to explain all options. The commit explains it and it can be easily looked up with git.

Reply:
Ah, finally I found what is that -a. It is `\-a\ (\-\-kexec-syscall-auto)` so long option is `--kexec-syscall-auto`.

About unstable I get this impression, because it is added just as patch to kexec and not as part of upstream tarball. But I see it now even in upstream one, so good.

https://git.kernel.org/pub/scm/utils/kernel/kexec/kexec-tools.git/tree/kexec/kexec.c?id=HEAD#n1010

well, lets keep it as NP to use long option and I approve it.